webauthn-authenticator-rs/src/error.rs

```diff
@@ -72,6 +72,10 @@ pub enum WebauthnCError {
     /// authenticator. You may need to set a PIN, or use a different
     /// authenticator.
     UserVerificationRequired,
+    /// The library is in an unexpected state. This could indicate that
```

Comment:
Is there a way we can dump or get more info if this happens?

Reply:
I have documentation in the `btle` branch for where this happens.